charts/gitprime-app/v4.1.0/questions.yml

```diff
@@ -0,0 +1,748 @@
+    description: The number of threads per job to use for the reprocess workers.
+    default: 2
+
+  - variable: dataPipeline.reprocessWorker.reprocessMaxCommitCount
+    type: string
+    required: true
+    group: "Data Pipeline / Reprocess Workers"
+    label: Reprocess Worker Maximum Commit Count
+    description: The maximum number of commits each worker will handle before resting.
+    default: 10000
+
+  - variable: dataPipeline.reprocessWorker.resources.requests.cpu
+    type: string
+    required: true
+    group: "Data Pipeline / Reprocess Workers"
+    label: Reprocess Worker Requested CPU
+    description: The number of CPUs the reprocess workers will request.
+    default: "2"
+
+  - variable: dataPipeline.reprocessWorker.resources.requests.memory
+    type: string
+    required: true
+    group: "Data Pipeline / Reprocess Workers"
+    label: Reprocess Worker Requested Memory
+    description: The maximum amount of memory a reprocess worker will request.
+    default: "4Gi"
+
+  - variable: dataPipeline.reprocessWorker.resources.limits.cpu
+    type: string
+    required: true
+    group: "Data Pipeline / Reprocess Workers"
+    label: Reprocess CPU Limit
+    description: The maximum number of CPUs the reprocess workers can use.
+    default: "4"
+
+  - variable: dataPipeline.reprocessWorker.resources.limits.memory
+    type: string
+    required: true
+    group: "Data Pipeline / Reprocess Workers"
+    label: Reprocess Worker Memory Limit
+    description: The maximum amount of memory a reprocess worker can use.
+    default: "8Gi"
+
+  - variable: dataPipeline.reprocessWorker.logLevel
+    type: enum
+    required: true
+    group: "Data Pipeline / Reprocess Workers"
+    label: Log Level
+    description: The level of logging for this deployment.
+    options:
+      - TRACE
+      - DEBUG
+      - INFO
+      - WARN
+      - ERROR
+    default: "INFO"
+
+  # BP Worker Settings
+  - variable: dataPipeline.bpWorker.replicaCount
+    type: string
+    required: true
+    group: "Data Pipeline / BP Workers"
+    label: Number of Pods for BP Workers
+    description: The number of pods to deploy for the workers that handle background processing.
+    default: 1
+
+  - variable: dataPipeline.bpWorker.javaOptions
+    type: string
+    required: true
+    group: "Data Pipeline / BP Workers"
+    label: BP Worker Java Options
+    description: The Java options for the BP workers.
+    default: "-d64 -Xms2g -Xmx4g"
+
+  - variable: dataPipeline.bpWorker.bpListeners
+    type: string
+    required: true
+    group: "Data Pipeline / BP Workers"
+    label: BP Worker Concurrent Jobs
+    description: The number of concurrent jobs for the BP workers.
+    default: 1
+
+  - variable: dataPipeline.bpWorker.bpThreadsPerListener
+    type: string
+    required: true
+    group: "Data Pipeline / BP Workers"
+    label: BP Worker Thread Count
+    description: The number of threads per job to use for the BP workers.
+    default: 1
```

Comment:
Does this configuration do anything?  I'm in favor of removing config options that do not have an impact on the application.  In this case, I don't think we use threads within BG processing.

Reply:
Removed.